anypoint-mq/4.x/modules/ROOT/pages/anypoint-mq-failover.adoc

```diff
@@ -31,13 +35,40 @@ To set this application property in Runtime Manager:
 
 . Sign in to Anypoint Platform and click *Runtime Manager* in the navigation menu.
 . Deploy a new app or change the settings for a deployed app.
-. On the *Properties* tab, specify values for the property:
+. On the *Properties* tab, specify the value for the property:
 +
-image::mq::mq-set-properties-failover-enabled.png["Properties tab with failover properties specified"]
+image::mq-set-properties-failover-enabled.png["Properties tab with failover system property set to true"]
 
 Alternatively, set the property in *Run Configurations* in Anypoint Studio.
 For more information, see xref:mule-runtime::mule-app-properties-system.adoc[System Properties in Mule Apps].
 
+[[disable-failover-feature]]
+== Disable Failover in Your App
+
+To disable the failover feature in your app, set the `cross.region.failover.feature.enabled` system property to `false`.
```

Comment:
I'm wondering if this section will cause confusion. When I read this, the first question that comes to my mind is if I can simply disable the feature from the app itself, then why do I need to turn off the feature from the UI.

When customers turn off the feature from the UI, they just need to restart their apps. They don't really need to set this system property to false. 
But leaving this system property as true will prevent them from consuming from the fallback queue directly. So customers need to turn off this system property for any application where they intend to consume from the fallback queue directly.
I think we should also add a screenshot to explain what "consume directly from fallback queue means" - specifying the fallback queue name as the "Destination" for Publish/Consume and as "Queue" for Subscribe

Reply:
@GouthamiKondakindi - is there any use case, besides consuming directly from the fb queue, for disabling failover on an app?

Reply:
Not really.. that's the main usecase.. 

We added that system property in the app as a safety mechanism.. if customers are running into some issue with the latest code we released, instead of rolling back to a previous version of connector, they can set this system property to false and we built the connector so that none of the cross region fallback code in the connector will be executed when the system property is false. But we don't need to document this :)